vspace/src/main/webapp/WEB-INF/views/staff/modules/sequences/sequence.jsp

```diff
@@ -1,23 +1,207 @@
-<%@ taglib uri="http://java.sun.com/jsp/jstl/functions" prefix="fn" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/functions" prefix="fn"%>
 <%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
 
 <ol class="breadcrumb">
-  <li class="breadcrumb-item"><a href="<c:url value="/staff/dashboard" />">Dashboard</a></li>
-  <li class="breadcrumb-item"><a href="<c:url value="/staff/module/list" />">Modules</a></li>
-  <li class="breadcrumb-item"><a href="<c:url value="/staff/module/${module.id}" />">${module.name}</a></li>
-  <li class="breadcrumb-item active">${sequence.name}</li>
+	<li class="breadcrumb-item"><a
+		href="<c:url value="/staff/dashboard" />">Dashboard</a></li>
+	<li class="breadcrumb-item"><a
+		href="<c:url value="/staff/module/list" />">Modules</a></li>
+	<li class="breadcrumb-item"><a
+		href="<c:url value="/staff/module/${module.id}" />">${module.name}</a></li>
+	<li class="breadcrumb-item active">${sequence.name}</li>
 </ol>
 
-<h1>Sequence: ${sequence.name}</h1>
+
+
+<script>
+
+
+$(document).ready(function() { 
+ 
+	
+    //-------- edit contentblock description Sequence starts --------
+    $("#submitDescription").hide()
+    $("#cancelEditDescription").hide()	
+    
+    $("#editDescription").click(function() {
+    	
+    	// using data-attributes
+    	var description = $("#description").data('value')  // gets value
+        $('<textarea id="newDescription" style="margin-top: 1%;" class="form-control" type="text" data-value="">'+description+'</textarea>').insertBefore( "#description" );
+        $("#description").hide()
+        $("#newDescription").val(description)
+        $("#newDescription").data('value',description);
+        $("#editDescription").hide()
+        $("#submitDescription").show()
+        $("#cancelEditDescription").show()
+        
+        
+    });
+
+    
+    // --- Submit Description ------
+    
+    $("#submitDescription").click(function() {
+        var formData = new FormData();
+        formData.append('description', $("#newDescription").val());
+        $.ajax({
+        url: "<c:url value="/staff/module/${module.id}/sequence/${sequence.id}/edit/description?${_csrf.parameterName}=${_csrf.token}" />",
+        type: 'POST',
+        cache       : false,
+        contentType : false,
+        processData : false,
+        data: formData,
+        enctype: 'multipart/form-data',
+        success: function(data) {
+            // replace text box with new description
+            $("#submitDescription").hide()
+            $("#cancelEditDescription").hide()
+            $("#editDescription").show()
+            var val = $("#newDescription").val();
+            $("#editDescription").show()
+            $("#description").show() 
+            $('#description').insertBefore( "#newDescription" );
+            $("#description").text(val);
+            $("#description").data('value',val);
+            $("#newDescription").hide()
```

Comment:
every time the user edits the description, you create a new textarea and add it to the html. that would result in many textareas that are not used, wouldn't it?

Reply:
Previously, the code was creating new description tag, while editing each time. Hence, as you suggested in the previous pull request to use the original description tag. I have reused the same. It's not creating a new one, just using the "description" tag initially created.

Reply:
line 32 creates a new textarea, doesn't it?

Reply:
Yes, because while editing we need a text-area to edit. As there is no already created text-area for description. Hence we are creating it. I will close it on submitting, to avoid multiple creations.